src/components/Documentation/RightPanel/index.tsx

```diff
@@ -78,7 +104,10 @@ const RightPanel: React.FC<IRightPanelProps> = ({
   }
 
   useEffect(() => {
-    const throttledSetCurrentHeader = throttle(updateCurrentHeader, 100)
+    const throttledSetCurrentHeader = (): void => {
+      throttle(updateCurrentHeader, 100)()
+      debounce(scrollSidebar, 100)()
```

Comment:
The extra `()` in `throttle(updateCurrentHeader, 100)()` looks weird, but that's because lodash throttle returns a function for invocation. I don't use lodash much, so at first this confused me, but thanks to VSCode tooling ❤️ I can easily see what going on under the hood.

![image](https://user-images.githubusercontent.com/22982964/124348093-93b05780-dc1a-11eb-9c23-d2fe2000248a.png)

Reply:
This doesn't quite work in its current form, as every time this `throttledSetCurrentHeader` is invoked it's using `throttle` and `debounce` to create a new function every time the regular non-throttled/debounced `throttledSetCurrentHeader` runs, so effectively neither child function actually gets limited.

I tested it out by cranking up the delay to something unreasonable and sticking `console.log`s in all the relevant functions, and the child functions are still being run every time the parent is.

Reply:
I ended up removing this afterwards in my private repo in my deeper fix.
https://github.com/kangzeroo/dvc.org/pull/1

Good catch, I should have read up into lodash docs more. Makes sense that the throttle should be on the outside. The PR in my private repo removed this extra function entirely, so the throttle is back to how it was before.